### ISSUE TITLE
Video capture session shows with delay on iPhone XR

### ISSUE THREAD

**Issue by Antalinka** [shogo4405/HaishinKit.swift#574]
Run example iOS, video capture shows with some delay on iPhone

**To Reproduce**
1. Run application
2. See capturing video

**Expected behavior**
Video should be shown without any delay

iOS 12.4, iPhone XR
Can't see this problem on iPhone 8

**Comment by Antalinka** [comment]
anyway it still exist

**Comment by spllr** [comment]
@Antalinka you are experiencing the Apple cinematic stabilisation mode.

Change `stream.captureSettings["preferredVideoStabilizationMode"] = AVCaptureVideoStabilizationMode.off.rawValue` or any other non cinematic mode and you should have realtime viewer again.

Appears `AVCaptureVideoStabilizationMode.off.rawValue` will result in cinematic stabilisation.

**Comment by nwparker** [comment]
> @Antalinka you are experiencing the Apple cinematic stabilisation mode.
> 
> Change `stream.captureSettings["preferredVideoStabilizationMode"] = AVCaptureVideoStabilizationMode.off.rawValue` or any other non cinematic mode and you should have a realtime viewer again.
> 
> Appears `AVCaptureVideoStabilizationMode.auto.rawValue` will result in cinematic stabilisation.

Hey @spllr, the issue I understood from @Antalinka's description is different than the one you have proposed a fix for. The fix you proposed is for lagging video (#536).
The issue that I undestood @Antalinka having (the one I also experience) is that there is a delay of the image shown on the recording screen and the image the phone looking at (in real-time). There are no lag spikes in the recording, it's just a constant input lag from the real world to the video recording screen.

@Antalinka  -- maybe you can expand to confrim this is your issue?

**Comment by spllr** [comment]
Hi @nwparker. Thanks for pointing out #536, I had a quick look and there might be a correlation, I am not sure.

As I understood the question from @Antalinka it indeed concerned a time difference between the lens and the viewer, not the actual produced video.

For instance, placing an item in front of the camera lens would only show up in the viewer a moment later, but the actual video stream was fine. After changing the video stabilization mode from cinematic (or auto) to off, the lag disappeared. You will also notice that the front-facing camera does not display the same issue.

In another camera project I worked on, without HaishinKit, we experienced similar behaviour.

NOTE: I have been working with  [SRTHaishinKit.swift](https://github.com/shogo4405/SRTHaishinKit.swift) which was build against an older version of HaishinKit (0.11.3), I have since upgraded SRTHaishinKit.swift to be compatible with current HEAD but have not checked if the same behaviour occurred when changing the video stabilization mode back to auto.

I can not confirm if there is not other issue at play, but in order to get a real-time viewer in the demo app, changing the stabilization mode should be sufficient.

Hope this answers your question.
